capella2polarion/converters/converter_config.py

```diff
@@ -308,14 +309,55 @@ def add_prefix(polarion_type: str, prefix: str) -> str:
     return polarion_type
 
 
+def _filter_converter_config(
+    config: dict[str, dict[str, t.Any]]
+) -> dict[str, dict[str, t.Any]]:
+    custom_converters = (
+        "include_pre_and_post_condition",
+        "linked_text_as_description",
+        "add_context_diagram",
+        "add_tree_diagram",
+        "add_jinja_fields",
+        "jinja_as_description",
+    )
+    filtered_config = {}
+    for name, params in config.items():
+        params = params or {}
+        if name not in custom_converters:
+            logger.error("Unknown converter in config %r", name)
+            continue
+
+        if name == "add_context_diagram":
+            params = _filter_context_diagram_config(params)
```

Comment:
You are only parsing the config for context_diagrams but not for tree_view diagrams. On the other hand the new `filters` property seems to be supported by the tree view renderer, too. Is this correct?